packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -720,31 +720,26 @@ export default abstract class BaseCall implements IWebRTCCall {
           parameters.encodings = [{ rid : 'h' }];
         }
         logger.info("Parameters: ", parameters);
+        logger.info("Setting max ", _kind === 'audio' ? "audio" : "video", " bandwidth to: ", max, " [bps]");
+
+        parameters.encodings[0].maxBitrate = max;
```

Comment:
I think it is a good idea to add a validation here if has `parameters.encodings[0]` and `parameters.encodings[0].maxBitrate` 

```
if (parameters.encodings[0] && parameters.encodings[0].maxBitrate) {
     parameters.encodings[0].maxBitrate = max;
}
```

Reply:
I do a check for it at line 719, just above:

```
if (!parameters.encodings) {
          parameters.encodings = [{ rid : 'h' }];
        }
```